scripts/macpython-install-python.sh

```diff
@@ -100,15 +96,13 @@ function fill_pyver {
     # Convert major or major.minor format to major.minor.micro
     #
     # Hence:
-    # 2 -> 2.7.11  (depending on LATEST_2p7 value)
-    # 2.7 -> 2.7.11  (depending on LATEST_2p7 value)
+    # 3 -> 3.11.0  (depending on LATEST_3p11 value)
```

Comment:
Should we change this to point to 3.13?

Reply:
No. We intentionally build against 3.11 as the latest.